ui/scripts/movie.js

```diff
@@ -109,6 +123,31 @@ export default class Movie {
         return conteiner;
     }
 
+    // expects countries object that has name and flag fields, class  
+    getFlags(counries, name) {
+        let flags = "";
+
+        if (counries.length !== 0) {
+            const list = document.createElement("ul");
+            list.classList.add(name);
+
+            counries.map(country => {
+                if (country.flag) {
+                    flags += `<li class="country-flag">${country.flag}<span class="country-name-tooltip">${country.name}</span></li>`;
+                } else {
+                    // handle null flag
+                    // &#127987; adds white flag symbol
+                    flags += `<li class="country-flag">&#127987;<span class="country-name-tooltip">${country.name}</span></li>`;
```

Comment:
These html character codes embedded in code are a bit concerning. Can we start defining them somewhere in a common module as variables with meaningful names, then refer here? (Same for right arrow merged in another PR.)

Reply:
We can do it as a separate PR. I'll add an issue for that and will work on it